src/stylesheets/_variables.scss

```diff
@@ -28,4 +29,6 @@ $margin-xlarge: 25px;
 $margin-xxlarge: 30px;
 $margin-xxxlarge: 35px;
 
-$standard-form-element-height: 38px
+$margin-form: 80px;
```

Comment:
This variable is too tightly couple. It can now only refer to the margin for your form and nothing else. Also, it doesn't specify which margin (top, left, right, bottom). A better structure could be to have various sizes defined:

```
$margin-sm-horizontal: 10px;
$margin-lg-horizontal: 20px;
$margin-xl-horizontal: 40px;
$margin-xxl-horizontal: 60px;
...
$margin-sm-vertical: 5px;
...
etc.
```